.github/workflows/check-layouts.yml

```diff
@@ -12,8 +12,9 @@ jobs:
   check-layouts:
     runs-on: ubuntu-latest
     steps:
-    - name: Checkout repo
-      uses: actions/checkout@v3
-    - run: python3 gen_layouts.py
-    - name: Check that the generated layouts.xml is uptodate, run python3 gen_layouts.py otherwise
-      run: git diff --exit-code
+      - name: Checkout repo
```

Comment:
What auto-formatter do you use for yaml ?

Reply:
I use Android Studio set to two spaces. Were tabs used before? I am confused.

Reply:
I did not enforce any style to the yml files before and was just curious.
The style you set is fine.

Reply:
Okay, that's good